module/applications/item/item-sheet-2.mjs

```diff
@@ -92,6 +92,55 @@ export default class ItemSheet5e2 extends ItemSheetV2Mixin(ItemSheet5e) {
         }));
       });
     }
+
+    if ( this.isEditable ) {
+      html.find(".activities .activity .name").on("click", this._onEditActivity.bind(this));
```

Comment:
It would probably still be good to be able to open the activity sheet even if the item is locked.